deliverables/README.md

```diff
@@ -2,8 +2,12 @@
 ###Intro to the Back End
 ####Required
 What is the difference between front- and back-end development?<br>
-Why do you need to install Ruby but not HTML/CSS?
+Front end devolopment deals with the things that the user can interact with within a website.
+Back end devolopment deals with the things that the user changes through interacting with what is made by front end devolopment.
+Examples being databases, servers or an application. 
 
+Why do you need to install Ruby but not HTML/CSS?
+The reason you need to install Ruby but not 
```

Comment:
Didn't finish this.  Basic idea is browser already knows how to interpret HTML, CSS, and Javascript so that's why you don't need to install an interpreter for these.  However most of the time you don't have a program that knows how to read ruby yet, so you need to install the ruby interpreter to run your ruby code.